src/test/ensemble_config_change_stress.cpp

```diff
@@ -1926,3 +2177,186 @@ TEST_F(StressModelConfigChanges, AddModelDuringGetModelStatusLoad) {
         requiredLoadResults,
         allowedLoadResults);
 }
+
+#if (MEDIAPIPE_DISABLE == 0)
+class StressMediapipeChanges : public StressPipelineConfigChanges {
+    const std::string modelName = "dummy";
+    const std::string modelInputName = "b";
+    const std::string modelOutputName = "a";
+
+public:
+    std::string getServableName() override {
+        return modelName;
+    }
+    void SetUp() override {
+        SetUpServerInstance(createStressTestPipelineOneDummyConfig());
+    }
+};
+TEST_F(StressMediapipeChanges, AddGraphDuringPredictLoad) {
+    // we add another definition during load
+    SetUpConfig(basicMediapipeConfig);
+    bool performWholeConfigReload = true;
+    std::set<StatusCode> requiredLoadResults = {StatusCode::OK};  // we expect full continuouity of operation
+    std::set<StatusCode> allowedLoadResults = {
+        StatusCode::MEDIAPIPE_EXECUTION_ERROR,                // TODO to remove after MP investigation
+        StatusCode::MEDIAPIPE_GRAPH_INITIALIZATION_ERROR,     // TODO to remove after MP investigation
```

Comment:
Correct me if I'm wrong but those statuses are in fact possible to get during custom MP pipeline exection, but our stress tests use well known predefined calculators and we are sure that we cannot get such statuses. Those would only appear in wrongly configured graphs or vulnerable calculators.
If so, I think we do not need to allow those statuses.

Reply:
You're correct. Those statuses were appearing because we missed fix in MediapipeGraphDefinition. There is already fix for that, and I'm removing those status codes.